CHANGELOG.md

```diff
@@ -3,6 +3,76 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.9.0
+
+_Jun 22, 2023_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 We release a new package `@mui/x-charts`.
+  This package aims at simplifying the integration of charts into your dashboards. 📊
+
+  ![image](https://github.com/mui/mui-x/assets/45398769/613bae2f-537d-4ec3-8046-591d1a504a72)
+
+  It already contains [lines](https://mui.com/x/react-charts/lines/), [bars](https://mui.com/x/react-charts/bars/), and [scatter](https://mui.com/x/react-charts/scatter/) charts with basic customization features.
+  Please, visit the new [documentation](https://mui.com/x/react-charts/) to know what it does, and open issues to get the feature you need to be implemented.
+
+- 🚀 Introduce the UTC and timezone support for pickers.
```

Comment:
@flaviendelangle Feel free to suggest a better highlight 🙈

Reply:
Maybe we can add a small GIF, something like:

![ezgif-2-92aed425de](https://github.com/mui/mui-x/assets/3309670/f8b552b0-9ec2-417b-b769-95740308291d)